pkgs/applications/misc/neo/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib, stdenv, fetchurl, ncurses }:
+
+stdenv.mkDerivation rec {
+  pname = "neo";
+  version = "0.6";
+
+  src = fetchurl {
```

Comment:
Could we use `fetchFromGitHub` here?

Reply:
@IvarWithoutBones we could, although using `git` repository as source means adding a dependency on autotools which we don't need with a release tarball. Let me know if this argument is not good enough, and I'll revise it.

Reply:
Using release tarballs is fine and common practice